app/(header-default)/dirigeants/[slug]/_component/sections/entreprise/dpo/section.tsx

```diff
@@ -0,0 +1,116 @@
+import { DataSection } from '#components/section/data-section';
+import { TwoColumnTable } from '#components/table/simple';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { ISession } from '#models/authentication/user/session';
+import { IUniteLegale } from '#models/core/types';
+import { getDPO } from '#models/dpo';
+import { formatIntFr } from '#utils/helpers';
+
+type IProps = {
+  uniteLegale: IUniteLegale;
+  session: ISession | null;
+};
+
+const DPONotFound = () => (
+  <>
+    <p>Aucun DPO trouvé pour cette entreprise.</p>
```

Comment:
```suggestion
    <p>Aucun DPO trouvé pour cette structure.</p>
```
can be an asso or and administration